README.md

```
@@ -1,2 +1,25 @@
+## How to install
+1. goto repo directory
+2. make build directory 'mkdir build'
+3. go there 'cd build'
+4. call 'cmake ..'
+5. compile code 'make -jN' where N is number of cores
+6. to use Bober apped build directory to library path (e.g., 'export LD_LIBRARY_PATH=~/bober/build:$LD_LIBRARY_PATH')
+# how to use
+You can use it in any macro in principle you should call '  new Bober::HistogramEditor(h, gPad);' to start editor. 'h' is your histogram (TH1 and TH2 are supported) and gPad is pointer to pad for this histogram.
+When you finish your setting up the histogram you can go to tab 'Super' and set the path to the xml-file with configuration. You can configure only one pad+histogram per single xml file.
+To apply settings you have to import them e.g.:
+```
```

Markdownlint (0.35.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)